src/main/java/picard/illumina/parser/readers/CbclReader.java

```diff
@@ -406,4 +409,7 @@ public int getHeaderSize() {
         return allTiles;
     }
 
+    public void clear() {
```

Comment:
Could this just be `close()`?  I dont' see it being used anywhere else but another `close()` call.